Applications/MaterialAssist/Client/MaterialAssistClientEdgebands.cs

```diff
@@ -0,0 +1,334 @@
+﻿using System.Text;
+
+using HomagConnect.Base.Services;
+using HomagConnect.MaterialAssist.Contracts.Base;
+using HomagConnect.MaterialAssist.Contracts.Edgebands;
+using HomagConnect.MaterialAssist.Contracts.Edgebands.Interfaces;
+using HomagConnect.MaterialManager.Contracts.Material.Edgebands;
+
+namespace HomagConnect.MaterialAssist.Client
+{
+    /// <summary>
+    /// materialAssist client for edgebands
+    /// </summary>
+    public class MaterialAssistClientEdgebands : ServiceBase, IMaterialAssistClientEdgebands
+    {
+        /// <inheritdoc />
+        public MaterialAssistClientEdgebands(HttpClient client) : base(client) { }
+
+        #region Delete
+
+        /// <inheritdoc />
+        public async Task DeleteEdgebandEntity(string id)
+        {
+            if (string.IsNullOrEmpty(id))
+            {
+                throw new ArgumentException("Id must not be null or empty", nameof(id));
+            }
+
+            var url = $"{_BaseRoute}?/delete?{_Id}={Uri.EscapeDataString(id)}";
+
+            await DeleteObject(new Uri(url, UriKind.Relative));
+        }
+
+        #endregion Delete
+
+        #region Private methods
+
+        private static List<string> CreateUrls(IEnumerable<string> codes, string searchCode, string route = "",
```

Comment:
CreateUrls is the same in all clients? Maybe you can create an extension that can be used by all clients?

Reply:
yes, I will probably adress this in a seperate PR in the future to not mess with the purpose of this one